[2023-05-05 05:28:52,506] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: postgres_pipeline.query_employees manual__2023-05-05T05:28:50.315153+00:00 [queued]>
[2023-05-05 05:28:52,515] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: postgres_pipeline.query_employees manual__2023-05-05T05:28:50.315153+00:00 [queued]>
[2023-05-05 05:28:52,516] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-05-05 05:28:52,516] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-05-05 05:28:52,517] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-05-05 05:28:52,528] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): query_employees> on 2023-05-05 05:28:50.315153+00:00
[2023-05-05 05:28:52,531] {standard_task_runner.py:52} INFO - Started process 6294 to run task
[2023-05-05 05:28:52,536] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_pipeline', 'query_employees', 'manual__2023-05-05T05:28:50.315153+00:00', '--job-id', '160', '--raw', '--subdir', 'DAGS_FOLDER/load_from_postgres.py', '--cfg-path', '/tmp/tmp7nibnfsp', '--error-file', '/tmp/tmp8vxmlo7h']
[2023-05-05 05:28:52,538] {standard_task_runner.py:77} INFO - Job 160: Subtask query_employees
[2023-05-05 05:28:57,662] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_pipeline.query_employees manual__2023-05-05T05:28:50.315153+00:00 [running]> on host ccbba2675476
[2023-05-05 05:28:57,716] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/xcom.py:323 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 05:28:57,743] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:126 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 05:28:57,753] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:162 SAWarning: Coercing Subquery object into a select() for use in IN(); please pass a select() construct explicitly
[2023-05-05 05:28:57,765] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:163 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 05:28:57,770] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgres_pipeline
AIRFLOW_CTX_TASK_ID=query_employees
AIRFLOW_CTX_EXECUTION_DATE=2023-05-05T05:28:50.315153+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-05T05:28:50.315153+00:00
[2023-05-05 05:28:57,791] {base.py:79} INFO - Using connection to: id: postgres_local. Host: host.docker.internal, Port: 5432, Schema: sample_source_db, Login: ***, Password: ***, extra: {}
[2023-05-05 05:28:57,804] {sql.py:375} INFO - Running statement: SELECT * FROM employees;, parameters: None
[2023-05-05 05:28:57,809] {sql.py:384} INFO - Rows affected: 10
[2023-05-05 05:28:57,840] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=postgres_pipeline, task_id=query_employees, execution_date=20230505T052850, start_date=20230505T052852, end_date=20230505T052857
[2023-05-05 05:28:57,875] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-05-05 05:28:57,933] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-05 05:28:57,935] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dag.py:1076 SADeprecationWarning: Query.value() is deprecated and will be removed in a future release.  Please use Query.with_entities() in combination with Query.scalar() (deprecated since: 1.4)
[2023-05-05 05:28:57,936] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dag.py:1076 SADeprecationWarning: Query.values() is deprecated and will be removed in a future release.  Please use Query.with_entities() (deprecated since: 1.4)
